src/web/core/screens/world-map-screen/world-map-screen.tsx

```diff
@@ -27,6 +31,11 @@ export const WorldMapScreen = () => {
 
     const watchId = navigator.geolocation.watchPosition(
       ({ coords: { latitude: lat, longitude: lng } }) => {
+        setAuthUserPosition({
+          lat,
+          lng,
+        })
```

Comment:
Maybe create a separate hook, that will track location?

Try to look up for the good hooks library.